src-plugins/manualRegistration/manualRegistrationToolBox.h

```diff
@@ -47,7 +47,7 @@ public slots:
     void reset();
 
 protected:
-    void hideEvent(QHideEvent *event);
+    virtual void clear();
```

Comment:
So, we are looking with @fcollot at the best way to use `virtual`. It seems that it's common to use `virtual` in derived classes because it improves the readability. You easily understand that this is a virtual function.